### PR TITLE
Create ElectrodoDoc.md

### PR DESCRIPTION
**Pull request by bismerk**
Signed-off-by: Evheny Merzliakov <bismerk0@gmail.com>

## Checklist for Hyperledger Lab Proposal PR

- [ ] Read [README.md](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/README.md) carefully
- [ ] Make sure your changes are committed with the proper sign-off
- [ ] Use the labs name as the PR title
- [ ] Copy the short description of the proposal as the description of the PR
- [ ] Remove this checklist section
- [ ] Submit your PR and the labs stewards will review your proposal
- [ ] Ask your sponsor to confirm sponsorship by stating so in a comment to the PR

### CODE REVIEW

File: labs/ElectrodoDoc.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- https://github.com/andrii482
+- https://github.com/alexshehovcov
+- https://github.com/Dmitrii-Kholodov
+- https://github.com/roman-kravchenko
+
+# Sponsor
+
+
+# Pre-existing repository
+- https://github.com/482solutions/ElectrodoDoc-meta
```
Comment (tkuhrt) [inline labs/ElectrodoDoc.md:27]: Please note that any repository that we transfer into Hyperledger Labs must meet the following criteria:
1. Be Apache 2.0 licensed
2. Have DCO sign-off for each of the commits. 

In looking at this repo, I see that neither of these are met. These will need to be fixed before we could transfer this repo to the hyperledger-labs organization.

File: labs/ElectrodoDoc.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: ElectdoroDoc
+[//]: Electrodo # (parent: Electrodo)
```
Comment (tkuhrt) [inline labs/ElectrodoDoc.md:4]:
```suggestion
parent: Labs
```

File: labs/ElectrodoDoc.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+---
+# Lab Name
+ElectdoroDoc
+
+# Short Description
+Electrodo - software service for climate-related assets and risk management to increase enterprise market value. Electrodo solves the problem of data/context integrity, data/context reliability, and trust for carbon-related assets lifecycle, climate-related risk management, and ESG reporting by using a standardized framework based on TCFD Recommendations and data verification on the blockchain.
+# Scope of Lab
+During the Prototype Challenge, we developed a prototype ElectrodoDoc (Fig. 1) that implements the basic functionality for the climate-related disclosure service based on Hyperledger Fabric, FIWARE context broker (Orion), and IPFS protocol. Our solution demonstrates ESG data/context management and self-verification principles in terms of climate-related disclosures' lifecycle: data/context collection and processing, data/context verification, and publication of sustainability report in line with TCFD Recommendations framework.
+![Fig 1.](https://user-images.githubusercontent.com/17604217/170219272-5d6c62ff-a3a0-4cac-940a-6959fe486981.png)
```
Comment (tkuhrt) [inline labs/ElectrodoDoc.md:13]: It would be great to add the PNG under `labs/images` folder.

**Review by tkuhrt — CHANGES_REQUESTED** [review]
See inline comments.

File: labs/ElectrodoDoc.md (hunk 1 above)
Comment (andrii482) [reply labs/ElectrodoDoc.md:27]: @tkuhrt
Hello. Is it okay, if DCO sign-off would be only in 'ElectrodoDoc-meta' repository (but not exist in other mentioned  in 'ElectrodoDoc-meta' repository repositories) ?

**Comment by bismerk** [comment]
Hi @tkuhrt, thank you for review, I have fix issues could you check the updates, and move forward?

File: labs/ElectrodoDoc.md (hunk 1 above)
Comment (tkuhrt) [reply labs/ElectrodoDoc.md:27]: I am not sure that I understand the question. There is a single repository listed above and as it currently stands, not all of the commits in this repository have DCO sign-offs. Before we can transfer this repo to the hyperledger-labs organization, we would need to have DCO sign-off completed.

Comment (tkuhrt) [reply labs/ElectrodoDoc.md:27]: I see that the commits have been changed and a license file added. Ignore my previous comment.

**Review by tkuhrt — APPROVED** [review]

**Review by andrii482 — APPROVED** [review]
ok

**Review by andrii482 — APPROVED** [review]

**Comment by andrii482** [comment]
@tkuhrt looks like there is some need for second approve from someone with write or admin permissions
image
image

**Review by vipinsun — APPROVED** [review]
lgtm

**Comment by vipinsun** [comment]
Approving this lab

**Comment by lehors** [comment]
Submit another PR with the needed changes.

**Pull request merged**